### PR TITLE
T43649

### PR DESCRIPTION
**Pull request by turrs**
please give review

https://www.loom.com/share/5a16f603bbc348da85215d0dcc19d446

### CODE REVIEW

File: src/Context/index.jsx [hunk 1]
```diff
@@ -48,9 +49,10 @@ export const ContextWrapper = props => {
         `users/${userId}/trainings/${paramsId}`,
         dataEdit
       );
-      Notification(messages);
+      Notification(messages, "success");
     } catch (error) {
-      console.log(error);
+      console.log(12222, error);
```
Comment (argorfc) [inline src/Context/index.jsx]: remove log

File: src/Context/index.jsx [hunk 2]
```diff
@@ -34,9 +34,10 @@ export const ContextWrapper = props => {
     try {
       var messages = "Event successfully created";
       const response = await Axios.post("/trainings", data);
-      Notification(messages);
+      Notification(messages, "success");
     } catch (error) {
-      console.log(error);
+      console.log(21111, error);
```
Comment (argorfc) [inline src/Context/index.jsx]: remove log

File: src/Pages/LoginPage/LoginPage.jsx [hunk 3]
```diff
@@ -61,10 +61,11 @@ export const LoginPage = ({ setToken }) => {
         localStorage.setItem("token", JSON.stringify(result.token));
         setToken(JSON.stringify(result.token));
         var messages = "Login success";
-        Notification(messages);
+        Notification(messages, "success");
         navigate("/");
       } catch (error) {
         console.log(error);
```
Comment (argorfc) [inline src/Pages/LoginPage/LoginPage.jsx]: remove log

File: src/Parts/FilterTrainingEvent/index.jsx [hunk 4]
```diff
@@ -26,13 +26,17 @@ const FilterTrainingEvent = () => {
 
   const onChangeSearching = value => {
     // setValueCardTraining(value);
```
Comment (argorfc) [inline src/Parts/FilterTrainingEvent/index.jsx]: remove unused line

**Review by argorfc — CHANGES_REQUESTED** [review]
Not approved.

File: src/Context/index.jsx (hunk 1 above)
Comment (turrs) [reply src/Context/index.jsx]: thankyou for review, this issue already fix on 4da229e93e8066f1e4b3e33ea8e2a40e32f9c6d0

File: src/Context/index.jsx (hunk 2 above)
Comment (turrs) [reply src/Context/index.jsx]: thankyou for review, this issue already fix on 4da229e93e8066f1e4b3e33ea8e2a40e32f9c6d0

File: src/Pages/LoginPage/LoginPage.jsx (hunk 3 above)
Comment (turrs) [reply src/Pages/LoginPage/LoginPage.jsx]: thankyou for review, this issue already fix on 4da229e93e8066f1e4b3e33ea8e2a40e32f9c6d0

File: src/Parts/FilterTrainingEvent/index.jsx (hunk 4 above)
Comment (turrs) [reply src/Parts/FilterTrainingEvent/index.jsx]: thankyou for review, this issue already fix on 4da229e93e8066f1e4b3e33ea8e2a40e32f9c6d0

**Pull request merged**